frontend/src/components/static_pages/about.css

```diff
@@ -11,15 +11,30 @@
 .about-people-row {
   margin-bottom: 20px;
 }
-
 .images {
-  margin: 0;
-  margin-bottom: 40px;
-  margin-left: 120px;
+ margin: 0;
+ margin-bottom: 40px;
+ margin-left: 120px;
 }
 
 .images img {
-  width: 300px;
+ width: 330px;
+ padding: 20px;
+ border: 1px solid #d3d3d3;
+}
+
+#kim-image {
+ margin-top: 60px;
```

Comment:
we need to just have this for non mobile environments. In mobile screens the image moves on top and this margin is causing a bit of an issue 
![image](https://user-images.githubusercontent.com/1637481/68438717-a72bc680-018a-11ea-8c9b-9793d6e6e44a.png)